sphinx/writers/latex.py

```diff
@@ -396,22 +398,27 @@ def __init__(self, document, builder):
                                     path.basename(builder.config.latex_logo)
         # setup multilingual package
         self.babel = ExtBabel(builder.config.language)
-        if self.elements['polyglossia']:
-            self.elements['babel'] = ''  # disable babel forcely
-            self.elements['multilingual'] = self.elements['polyglossia']
-        elif self.elements['babel']:
-            self.elements['multilingual'] = self.elements['babel']
-            self.elements['classoptions'] += ',' + self.babel.get_language()
-            if builder.config.language:
-                if not self.babel.is_supported_language():
-                    self.builder.warn('no Babel option known for language %r' %
-                                      builder.config.language)
-                self.elements['shorthandoff'] = self.babel.get_shorthandoff()
-                self.elements['fncychap'] = '\\usepackage[Sonny]{fncychap}'
+        if builder.config.language:
+            if not self.babel.is_supported_language():
+                self.builder.warn('no Babel option known for language %r' %
+                                  builder.config.language)
+                self.elements['classoptions'] += ',english'
+                self.elements['multilingual'] = self.elements['babel']
```

Comment:
Babel would be used when unsupported language is specified even if you use xelatex.